convertors.py

```diff
@@ -139,7 +141,7 @@ def ConvertFibitHRPoint(self, date, data_point):
 		return dict(
 			dataTypeName='com.google.heart_rate.bpm',
 			startTimeNanos=epoch_time_nanos,
```

Comment:
com.google.heart_rate.bpm should not specify a startTimeNanos (or it should be equal to the end time)

Reply:
Oh, we do have docs for this: https://developers.google.com/fit/datatypes/body

Reply:
I didn't have any success _omitting_ the start time, despite modifying remote.py, the API seemed to reject it, so I made it the same for the sample types (HR, weight, body fat?)

Annoyingly Fitbit seems to give weight as 23:59 on the day you enter it, regardless of the actual time you do so...